<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0 ,y2:1,stop:0 rgba(255, 255, 255, 255), stop:0.5 rgba(214,214,255,255), stop:1 rgba(255,255,255,255));</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="6">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_4">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QLabel" name="destinatario">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>400</width>
              <height>30</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(236, 243, 253);
border-radius: 10px;</string>
            </property>
            <property name="text">
             <string>Para:</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="escribir_mensaje">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>330</y>
              <width>400</width>
              <height>70</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
           </widget>
           <widget class="QPushButton" name="enviar">
            <property name="geometry">
             <rect>
              <x>330</x>
              <y>420</y>
              <width>75</width>
              <height>24</height>
             </rect>
            </property>
            <property name="text">
             <string>Enviar</string>
            </property>
           </widget>
           <widget class="QPushButton" name="zumbido">
            <property name="geometry">
             <rect>
              <x>380</x>
              <y>300</y>
              <width>30</width>
              <height>30</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(236, 243, 253);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>zumbido.jpg</normaloff>zumbido.jpg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
           <widget class="QLabel" name="label_7">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>300</y>
              <width>400</width>
              <height>30</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(236, 243, 253);
border-radius: 10px;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="label_10">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>370</y>
              <width>400</width>
              <height>20</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(236, 243, 253);
border-radius: 10px;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QPlainTextEdit" name="plainTextEdit">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>50</y>
              <width>400</width>
              <height>200</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
           </widget>
           <widget class="QFrame" name="frame_5">
            <property name="geometry">
             <rect>
              <x>460</x>
              <y>20</y>
              <width>232</width>
              <height>443</height>
             </rect>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QLabel" name="fotoperfil1">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>50</y>
               <width>180</width>
               <height>120</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QLabel" name="fotoperfil2">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>270</y>
               <width>180</width>
               <height>120</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QLabel" name="nombre1">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>20</y>
               <width>180</width>
               <height>30</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(236, 243, 253);
</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
            <widget class="QLabel" name="nombre2">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>240</y>
               <width>180</width>
               <height>30</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(236, 243, 253);</string>
             </property>
             <property name="text">
              <string>Tu</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
            <widget class="QLabel" name="label_8">
             <property name="geometry">
              <rect>
               <x>15</x>
               <y>16</y>
               <width>190</width>
               <height>181</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(236, 243, 253);
border-radius: 10px;
</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
            <widget class="QLabel" name="label_9">
             <property name="geometry">
              <rect>
               <x>15</x>
               <y>230</y>
               <width>190</width>
               <height>181</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(236, 243, 253);
border-radius: 10px;
</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
            <widget class="QFrame" name="frame_3">
             <property name="geometry">
              <rect>
               <x>-470</x>
               <y>-20</y>
               <width>698</width>
               <height>74</height>
              </rect>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <widget class="QPushButton" name="cfoto">
              <property name="geometry">
               <rect>
                <x>540</x>
                <y>40</y>
                <width>111</width>
                <height>24</height>
               </rect>
              </property>
              <property name="text">
               <string>CAMBIAR FOTO</string>
              </property>
             </widget>
            </widget>
            <zorder>label_9</zorder>
            <zorder>label_8</zorder>
            <zorder>fotoperfil1</zorder>
            <zorder>fotoperfil2</zorder>
            <zorder>nombre1</zorder>
            <zorder>nombre2</zorder>
            <zorder>frame_3</zorder>
           </widget>
           <widget class="QComboBox" name="listaamigos">
            <property name="geometry">
             <rect>
              <x>308</x>
              <y>260</y>
              <width>101</width>
              <height>22</height>
             </rect>
            </property>
            <item>
             <property name="text">
              <string>Todos</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Ricardo</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Tigre</string>
             </property>
            </item>
           </widget>
           <zorder>label_7</zorder>
           <zorder>destinatario</zorder>
           <zorder>escribir_mensaje</zorder>
           <zorder>enviar</zorder>
           <zorder>zumbido</zorder>
           <zorder>label_10</zorder>
           <zorder>plainTextEdit</zorder>
           <zorder>frame_5</zorder>
           <zorder>listaamigos</zorder>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
